lookup/fileenv.go

```diff
@@ -119,6 +118,23 @@ func NewFileEnvLookup(file string, parent config.EnvironmentLookup) (*FileEnvLoo
 		}
 	}
 
+	parent := qLookup.parent
+	qVariables := qLookup.variables
+	questions := qLookup.questions
+
+	combMap := map[string]string{}
+	combMap = utils.MapUnion(variables, qVariables)
+
+	for key, question := range questions {
+		if _, ok := combMap[key]; ok {
+			continue
+		}
+		answer := ask(question)
+		if answer != "" {
+			variables[key] = answer
+		}
+	}
+
```

Comment:
Can you give a brief description on why this change would fix the issue?

Reply:
@StrongMonkey you first lookup rancher-compose to gather all the questions, then lookup env-file to parsing those key-value pairs, after that, iterate over all questions to see whether that question key has a value being parsed, if not, simply ask user to provide one